From e2396fc8a202f5f12a44a1a4f2075c1106fe656c Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Thu, 20 Mar 2014 16:41:52 +0400
Subject: [PATCH] DON'T MERGE - add patch that fixes U-Boot host tools building

In U-Boot 2014.04 host tools building is sort of broken now.
One needs:
 1) Config an existing board
 2) Have corresponding to (1) cross tools in path

Even though logically host tools has nothing to do with target (cross) tools.
As soon as mainline U-Boot is fixed remove this patch.

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index 4657577..fad56ba 100644
--- a/config.mk
+++ b/config.mk
@@ -58,7 +58,7 @@ PLATFORM_CPPFLAGS += -pipe
 LDFLAGS += $(PLATFORM_LDFLAGS)
 LDFLAGS_FINAL += -Bstatic
 
-export PLATFORM_CPPFLAGS
+export CROSS_COMPILE=
 export RELFLAGS
 export LDFLAGS_FINAL
 export CONFIG_STANDALONE_LOAD_ADDR
-- 
1.8.5.3

